### DISCUSSION TITLE
List of test cases for duckdb syntax

### DISCUSSION THREAD

**Discussion by cookiejest** [duckdb/duckdb#5842] [Q&A]
Hi, we are trying to train an AI model to generate some SQL statements for us, but we noticed some small differences between duckdb functions and standard postgresql (e.g split_part). Is there a place we can get a list of test cases for duckdb SQL statements?

**Comment by Mytherin** [comment]
The [test folder](https://github.com/duckdb/duckdb/blob/master/test/sql/function/string/test_string_split.test) is filled with SQL tests with their expected outputs.

If you notice minor differences between Postgres and DuckDB we would also generally be happy to review a PR to address those differences/add extra aliases so that DuckDB's behavior matches that of Postgres.
